customManager/EventManager.ts

```
@@ -0,0 +1,50 @@
+    eventList.push(fn);
+    this.eventList[event] = eventList;
+  }
+
+  off(event: string, fn: Function) {
+    const eventList = this.eventList[event] || [];
+    eventList.splice(eventList.indexOf(fn), 1);
+    this.eventList[event] = eventList;
+  }
+
+  once(event: string, fn: Function) {
+    const eventList = this.eventList[event] || [];
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)